client/my-sites/themes/controller.jsx

```diff
@@ -6,11 +6,13 @@ import React from 'react';
 /**
  * Internal Dependencies
  */
-import SingleSiteComponent from './single-site';
-import MultiSiteComponent from './multi-site';
+import SingleSiteComponent from 'my-sites/themes/single-site';
+import MultiSiteComponent from 'my-sites/themes/multi-site';
```

Comment:
I had to make the module names more explicit so that these could be stubbed out by webpack's `NormalModuleReplacementPlugin`. Without the full path, I'd be stubbing out any modules matching `./single-site` or `./multi-site` which is hazardous if other sections of the app have similarly-named modules. This lets me replace just the modules under `my-sites/themes`.